Core/tests/WholeProjectRefactorer.cpp

```diff
@@ -106,14 +108,26 @@ const std::vector<const gd::EventsList *> GetEventsLists(gd::Project &project) {
   return eventLists;
 }
 
+const std::vector<const gd::ObjectsContainer *> GetObjectContainers(gd::Project &project) {
```

Comment:
Should the name be more specific?

```suggestion
const std::vector<const gd::ObjectsContainer *> GetObjectContainersForMyOtherEventsBasedObject(gd::Project &project) {
```

Reply:
No, I should remove this method as I don't use it.
I though I would factorize tests for layout and ebo but it's not actually possible.